sync_check_terraform/service/docker-compose.yml

```diff
@@ -94,6 +94,7 @@ services:
     depends_on:
       - forest_mainnet
       - forest_calibnet
+    restart: unless-stopped
```

Comment:
Why?

Reply:
using this we can restart the sync check after droplet reboots, docker service is restarted on droplet reboot which starts `forest-calibnet`, `forest-mainnet` and `forest-tester` and sync check restarts on droplet reboot

Reply:
What about the logic contained in the `run_service.sh`, including the `sync_check_process.rb`? Will this process, monitoring disk usage, be spawned upon reboot?

Reply:
as per my understanding, we don't need to run `sync_check_process.rb` on reboot instead `forest-tester` commands get re-run on restart of docker services which include `sync_check.rb forest-calibnet` and `sync_check.rb forest-mainnet`

Reply:
Why don't we need to run it after a reboot? Say the machine gets rebooted - we still want to monitor disk usage and do the cleanup, no? This logic is not contained in the `forest-tester`.

While we do have garbage collection, we still need to do a cleanup from time to time (once a month or so, from what I see).

Reply:
if we re-run `run_service.sh` at reboot it will cleanup the disk with the `sync_check_process.rb`.

I don't quite get why we moved away from the original Docker container - it had restarts in-built. :)

Alternatively we can have a restart service that you introduced before, just not restarting docker but the `run_service.sh`.

Did I answer your question?

Reply:
yes

Reply:
done